### DISCUSSION TITLE
gear ratio modification DDT4ALL

### DISCUSSION THREAD

**Discussion by KZozo70** [cedricp/ddt4all#1119] [Q&A]
Hello,

I have a 2009 Laguna 3 Coupe 2.0 dci.
I had to change the gearbox in it and I couldn't find the same gearbox as the factory one.
Since then it has not worked without cruise control.
Is there any way and possibility to change the gear ratio with DDT4ALL?

Hi,
Zoli

**Comment by Marianpol** [comment]
Nope. Use Autotuner, Flex or any other tool to download ECU software and edit it.
